packages/wdio-cli/src/utils.ts

```diff
@@ -218,11 +218,11 @@ export function convertPackageHashToObject(pkg: string, hash = '$--$'): Supporte
     }
 }
 
-export async function renderConfigurationFile (answers: ParsedAnswers) {
+export async function renderConfigurationFile (answers: ParsedAnswers, configPath = '') {
     const tplPath = path.join(__dirname, 'templates/wdio.conf.tpl.ejs')
     const filename = `wdio.conf.${answers.isUsingTypeScript ? 'ts' : 'js'}`
     const renderedTpl = await renderFile(tplPath, { answers })
-    return fs.promises.writeFile(path.join(process.cwd(), filename), renderedTpl)
+    return fs.promises.writeFile(path.join(process.cwd(), configPath, filename), renderedTpl)
```

Comment:
```suggestion
    return fs.promises.writeFile(parsedAnswers.tsConfigFilePath, renderedTpl)
```

Reply:
@christian-bromann  parsedAnswers.tsConfigFilePath will have somepath/tsconfig.json , how will we pass this to write File of config file ? i think we need to keep the path.join here

Reply:
ah right, nevermind then